dashboard/src/utils/dateFunctions.js

```diff
@@ -1,11 +1,30 @@
+/**
+ * Bump Todate ahead by specified number of days
+ * @function
+ * @param {Date} date - Date to be bumped
+ * @param {number} numberOfDays - Number of days the date is to be bumped
+ * @returns {Date} - New Bumped date
+ */
 export const bumpToDate = (date, numberOfDays) => {
   return date.setUTCDate(date.getUTCDate() + numberOfDays);
 };
 
+/**
+ * Takes a string date and convert it into UTC Date object.
+ * Date should not contain timezone related information,else + "Z" will result in an error
+ * @function
+ * @param {string} strDate - Date in string format
+ * @returns {Date} - UTC Date object
+ */
 export const dateFromUTCString = (strDate) => {
   return new Date(strDate + "Z");
 };
 
+/**
+ * Return Date object set to midnight UTC
```

Comment:
Perhaps something more like, "Return a Date object representing the current day, UTC, with the time set to midnight."  (Your description doesn't mention _which_ date is returned.)

Likewise you might want to add the word "current" to the return value description at line 26.